developer_defaults.tf

```diff
@@ -167,6 +167,11 @@ variable "survey_runner_min_tasks" {
   default     = "1"
 }
 
+variable "respondent_account_url" {
+  description = "The url for the respondent log in that will be used to navigate to preprod"
```

Comment:
This description isn't quite right - the `that will be used to navigate to preprod` bit should be dropped.